sdk/core/Azure.Core.TestFramework/src/RecordedTestSanitizer.cs

```diff
@@ -33,7 +37,24 @@ public virtual void SanitizeHeaders(IDictionary<string, string[]> headers)
 
         public virtual string SanitizeTextBody(string contentType, string body)
         {
-            return body;
+            if (JsonPathSanitizers.Count == 0)
+                return body;
+            try
+            {
+                var jsonO = JObject.Parse(body);
+                foreach (var jsonPath in JsonPathSanitizers)
```

Comment:
Nit: not supposed to use `var`, though allows for the lvalue of instantiation expressions.

Reply:
updated